### ISSUE TITLE
Rename resources os.groupadd and os.useradd

### ISSUE THREAD

**Issue by gbraekmans** [jtopjian/waffles#186]
To me, it seems these are better named os.group and os.user.

```
os.groupadd  --name "ftp" --state absent
```

is considerably less readable than

```
os.group  --name "ftp" --state absent
```

**Comment by jtopjian** [comment]
Yup, good point. This is related to #6. `useradd` and `groupadd` are supposed to be for most (all?) Linux-based systems. I wanted to leave implementation open to other user/group management systems and possibly have a meta resource to autodetect which one to use underneath.

**Comment by gbraekmans** [comment]
I see. What about os.linux_user or maybe linux.user?

**Comment by jtopjian** [comment]
> What about os.linux_user or maybe linux.user

Those aren't bad alternative names. Let me mull this one over.
